DependencyInjection/HttplugExtension.php

```diff
@@ -56,6 +61,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_journal_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.plugin.history');
```

Comment:
I'll make this service "internal" instead.. Im using service id `httplug.collector.history_plugin`